programs/launcher/src/vsx_widget_launcher.h

```diff
@@ -137,6 +137,7 @@ class vsx_widget_launcher : public vsx_widget
     application_selection->add_option( 0, "VSXu Player" );
     application_selection->add_option( 1, "VSXu Artiste" );
     application_selection->add_option( 2, "VSXu Profiler" );
+    application_selection->add_option( 3, "VSXu Server" );
```

Comment:
This needs to be in #ifdef PLATFORM_LINUX or some such thing, as server is currently not supported on windows.